octoprint_TpLinkAutoShutdown/__init__.py

```diff
@@ -167,6 +190,27 @@ def get_settings_defaults(self):
 					auto=False,
 					movieDone=False,
 				),
+    			deviceFour=dict(
```

Comment:
Hi again here could you please check your indentation here. This needs to be indented once more for the deviceFour dict. Thanks

Reply:
Yeah not sure how that indent got borked. Fixed.